src/components/Editor/Editor.css

```diff
@@ -98,7 +98,7 @@ html[dir="rtl"] .editor-mount {
 }
 
 .highlight-line .CodeMirror-line {
-  animation: fade-highlight-out 1.5s normal forwards;
+  animation: fade-highlight-out 3s normal forwards;
```

Comment:
It seems a line would only be applied with the styles of `.highlight-line .CodeMirror-line` when navigating from the console. The navigation triggers the function at https://github.com/devtools-html/debugger.html/blob/master/src/components/Editor/index.js#L599.

The line is first cleared and highlighted under these 2 conditions: https://github.com/devtools-html/debugger.html/blob/master/src/components/Editor/index.js#L333-L349